fenix/app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -598,43 +522,10 @@ class BookmarksTest {
         }
     }
 
-    @Test
-    fun multipleBookmarkDeletionsTest() {
-        homeScreen {
-        }.openThreeDotMenu {
-        }.openBookmarks {
-            createFolder("1")
-            getInstrumentation().waitForIdleSync()
-            createFolder("2")
-            getInstrumentation().waitForIdleSync()
-            createFolder("3")
-            getInstrumentation().waitForIdleSync()
-        }.openThreeDotMenu("1") {
-        }.clickDelete {
-            verifyDeleteFolderConfirmationMessage()
-            confirmDeletion()
-            verifyDeleteSnackBarText()
-        }.openThreeDotMenu("2") {
-        }.clickDelete {
-            verifyDeleteFolderConfirmationMessage()
-            confirmDeletion()
-            verifyDeleteSnackBarText()
-            verifyFolderTitle("3")
-            // On some devices we need to wait for the Snackbar to be gone before continuing
-            TestHelper.waitUntilSnackbarGone()
-        }.closeMenu {
-        }
-
-        homeScreen {
-        }.openThreeDotMenu {
-        }.openBookmarks {
-            verifyFolderTitle("3")
-        }
-    }
-
+    // TestRail link: https://testrail.stage.mozaws.net/index.php?/cases/view/325639
     @SmokeTest
     @Test
-    fun changeBookmarkParentFolderTest() {
+    fun createBookmarkFolderTest() {
```

Comment:
let's remove the Smoke tag from this one as well

Reply:
Done !